manifests/secondarydns.yaml

```diff
@@ -143,3 +143,18 @@ spec:
           name: secondary-dns
       - name: secdns-zones
         emptyDir: {}
+---
+apiVersion: kubevirt.io/v1
```

Comment:
Is there a risk (or even just redundant) that it is already created when 
deployed as part of CNAO / HCO (or D/S) ?
If so, we will need to create it only where needed

Reply:
I will add it as part of `cluster-sync`. If ksd is deployed by CNAO/HCO then let's them to decide whether the fg is enabled or not.